index.js

```diff
@@ -150,7 +155,7 @@ function filter(patterns, opts) {
 
     fp = utils.unixify(fp, opts);
     while (i < len) {
-      var fn = matcher(patterns[i++], opts);
+      var fn = patterns[i++];
       if (!fn(fp)) {
         res = false;
```

Comment:
No substantial objections to the change, but if there's no change to API footprint why not bump patch instead of minor?

Also would still like to see tests added for the backslash scenarios, and generally it's safer to run anymatch and chokidar tests on any patch before release just in case.